packages/tpetra/core/example/BlockCrs/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+
+# This example works whether or not MPI is enabled.
+# It does not refer to MPI explicitly.
+TRIBITS_ADD_EXECUTABLE_AND_TEST(
+  BlockCrsPerfTest
+  SOURCES driver.cpp
+  ARGS 
+  COMM  mpi
```

Comment:
mpi is required for the test via Teuchos wrapper.

Reply:
You can set up you exectuable using Teuchos::GlobalMPISession to work with MPI and non-MPI builds.  See examples in the Teuchos, Thyra, and other packages.

It is better if you can write your tests so that they run with MPI and serial so you can remove the `COMM` argument altogether.